app/(admin)/admin/notifications/page.tsx

```diff
@@ -38,33 +48,111 @@ const AdminNotifications = (): JSX.Element => {
+
   return (
-    <section data-testid="admin-notifications-content">
-      <Button
-        label="Email Testers"
-        type="button"
-        onClick={participantsEmail}
-        data-testid="email-testers"
-      />
-      <form onSubmit={handleSubmit}>
-        <Label htmlFor="subject">Subject:</Label>
-        <Input
-          type="text"
-          id="subject"
-          data-testid="subject-text"
-          onChange={(e) => setSubject(e.target.value)}
-        />
-        <Label htmlFor="content">Content:</Label>
-        <textarea
-          name="content"
-          id="content"
-          onChange={(e) => setContent(e.target.value)}
-          data-testid="content-text"
+    <section
+      className="flex flex-col space-y-6"
+      data-testid="admin-notifications-content"
+    >
+      <p>
+        Choose the users you would like to email in{' '}
+        <span className="font-bold text-orange-500">{leagueName}</span>.
+      </p>
+      <RadioGroupDefault
+        defaultValue="all users"
+        onValueChange={handleRadioChange}
+        required
+      >
+        <div className="flex items-center space-x-2">
+          <RadioGroupDefaultItem
+            value="all users"
+            id="all"
+            data-testid="all-users-option"
+          />
+          <LabelText htmlFor="all">All users</LabelText>
+        </div>
+        <div className="flex items-center space-x-2">
+          <RadioGroupDefaultItem
+            value="all survivors"
+            id="survivors"
+            data-testid="only-survivors-option"
+          />
+          <LabelText htmlFor="survivors">Only the survivors</LabelText>
+        </div>
+        <div className="flex items-center space-x-2">
+          <RadioGroupDefaultItem
+            value="all losers"
+            id="losers"
+            data-testid="only-losers-option"
+          />
+          <LabelText htmlFor="losers">Only the losers</LabelText>
+        </div>
+      </RadioGroupDefault>
+      <form
+        onSubmit={handleSubmit}
+        className="flex flex-col space-y-6 max-w-[80ch]"
+      >
+        <div className="flex gap-2 flex-col">
+          <LabelText htmlFor="subject" className="text-lg">
+            Subject:
+          </LabelText>
+          <Input
+            data-testid="subject-text"
+            id="subject"
+            name="subject"
+            onChange={(e) => setSubject(e.target.value)}
+            type="text"
+          />
+        </div>
+        <div className="flex gap-2 flex-col">
+          <LabelText htmlFor="content" className="text-lg">
+            Message:
+          </LabelText>
+          <Textarea
+            data-testid="content-text"
+            id="content"
+            name="content"
+            onChange={(e) => setContent(e.target.value)}
+          />
+        </div>
+        <p>
+          This email will be sent to{' '}
+          <span className="font-bold text-orange-500">
+            {emailRecipients.toLowerCase()}
```

Comment:
> Why not have this as `groupUsers` and map it out?

@alexappleget this isn't a bad idea, but I feel like it could also get to be a lot based on the amount of people that can be in a league. Also, is it really that important for the person to know exactly who they're emailing? Or instead, is it more important to know _which users_ they are emailing? I think the latter, which is why I went about it this way.